### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by Bestbynature**
## Features included in this pull request are as follows:

- [x] RSpec was added as a gem to the project.
- [x] A Request spec file was created for all the controllers.
- [x] For each action, the following were checked:

- If the response status was correct.
- If a correct template was rendered.
- If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    it 'returns the appropriate placeholders' do
+      get '/users/:user_id/posts/show'
+
+      expected_content = 'This is a particular post with the given ID out of all ' \
+                         'the posts for a given user'
+
+      expect(response.body).to include(expected_content)
+    end
+  end
+end
```
Comment (mckent05) [inline spec/requests/posts_spec.rb:31]: - You have done well so far, but according to the project requirements, you are also required to test if the right template was rendered. For example `expect(response).to render_template(:show)` to test if it renders the `post#show` page. Please fix this for all test including `users_spec` 🙏

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      get '/users/:user_id'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'displays appropriate placeholder' do
+      get '/users/:users_id'
+      expect(response.body).to include('This is a particular user with the given id')
+    end
+  end
+end
```
Comment (mckent05) [inline spec/requests/users_spec.rb:27]: - According to the project requirements, please test if the right page was rendered, please fix this 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Bestbynature, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job testing your controllers 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/posts_spec.rb (hunk 1 above)
Comment (Bestbynature) [reply spec/requests/posts_spec.rb:31]: I appreciate you @mckent05. Thank you so much

File: spec/requests/users_spec.rb (hunk 2 above)
Comment (Bestbynature) [reply spec/requests/users_spec.rb]: I appreciate you @mckent05. Thank you so much

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Bestbynature,

✔️ Well-written tests

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**